src/legacy/components/LoadingScreen.tsx

```diff
@@ -144,6 +144,9 @@ class LoadingScreen extends Component<any, any> {
       })
       return true
     } catch (err) {
+      if (nRetry < 10) {
```

Comment:
I think this will make the ensurePastelParams to run 10 times even if all parameters are valid.

Reply:
Why? If they are valid, they will will never run into this `catch` it will be the above `return true`